docker/cloudberry_tests/scripts/tests/backup_after_failed_backup_test.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e -x
+
+CONFIG_FILE="/tmp/configs/backup_after_failed_config.json"
+COMMON_CONFIG="/tmp/configs/common_config.json"
+TMP_CONFIG="/tmp/configs/tmp_config.json"
+cat ${CONFIG_FILE} > ${TMP_CONFIG}
+echo "," >> ${TMP_CONFIG}
+cat ${COMMON_CONFIG} >> ${TMP_CONFIG}
+/tmp/pg_scripts/wrap_config_file.sh ${TMP_CONFIG}
+source /tmp/tests/test_functions/util.sh
+
+bootstrap_gp_cluster
+sleep 3
+setup_wal_archiving
+
+# init tables heap, ao, co
+insert_data
+
+# imitate failed backup (label='abc', fast=true, exclusive=true)
+psql -p 7000 -c "SELECT pg_start_backup('abc', true, true);"
+
+# run backup
+run_backup_logged ${TMP_CONFIG} ${PGDATA}
+
+stop_and_delete_cluster_dir
+
+# show the backup list
+wal-g backup-list --config=${TMP_CONFIG}
+
+# show the storage objects (useful for debug)
+wal-g st ls -r --config=${TMP_CONFIG}
+
+# it looks like a bug in `tryExtractFiles` - it can override files when WALG_DOWNLOAD_CONCURRENCY != 1
+# https://github.com/wal-g/wal-g/blob/master/internal/extract.go#L201-L253
+WALG_DOWNLOAD_CONCURRENCY=1  wal-g backup-fetch LATEST --in-place --config=${TMP_CONFIG}
```

Comment:
Here we have an issue with 'backup_label's:

in `part_001.tar.lz4` something like
```
START WAL LOCATION: 0/C000028 (file 000000010000000000000003)
CHECKPOINT LOCATION: 0/C000060
BACKUP METHOD: pg_start_backup
BACKUP FROM: primary
START TIME: 2024-10-15 19:38:24 UTC
LABEL: abc
START TIMELINE: 1
```

in `part_003.tar.lz4` something like
```
START WAL LOCATION: 0/C000028 (file 000000010000000000000003)
CHECKPOINT LOCATION: 0/C000060
BACKUP METHOD: streamed
BACKUP FROM: primary
START TIME: 2024-10-15 19:46:14 UTC
LABEL: 2024-10-15 19:46:13.608579 +0000 UTC m=+0.212280844
START TIMELINE: 1
```

as a result we getting
```
requested recovery stop point is before consistent recovery point
```